nimble/controller/src/ble_ll_conn_hci.c

```diff
@@ -200,7 +200,10 @@ ble_ll_conn_comp_event_send(struct ble_ll_conn_sm *connsm, uint8_t status,
             enh_ev->conn_itvl = htole16(connsm->conn_itvl);
             enh_ev->conn_latency = htole16(connsm->slave_latency);
             enh_ev->supervision_timeout = htole16(connsm->supervision_tmo);
-            enh_ev->mca = connsm->master_sca;
+            if (connsm->conn_role == BLE_LL_CONN_ROLE_MASTER)
```

Comment:
I think that since event is zeroed we can just set it for slave eg:

if (connsm->conn_role == BLE_LL_CONN_ROLE_SLAVE) {
    enh_ev->mca = connsm->master_sca;
}


also note that our coding style require {} also single line ifs